lecture2.tex

```diff
@@ -215,15 +233,15 @@ \section{Kompositioner}
     Metoden för det här beviset brukar kallas för ett pinnar-och-stjärnor-argument.\sidenote[][]{Engelska \emph{stars and bars argument}.}
 
     Vi studerar ord ur alfabetet $\{*,|\}$ med $k-1$ stycken pinnar och $n$ stycken stjärnor. Till exempel om $n=6$ och $k=5$
-    $$**||*|*|**.$$
+    $$**||*|*|**$$
 
     Vi tolkar detta ordet som en fördelning av föremål till personer såsom följer: Person ett får alla  stjärnor innan första strecket, person två alla mellan första och andra strecket, och så vidare, tills person $k$ får alla stjärnor efter det sista strecket. I vårt exempel får alltså person ett två föremål, person två inga, person tre och fyra ett var, och person fem får två.
 
     Detta etablerar en bijektion mellan våra ord och fördelningarna till personer. Vi vet från Proposition \ref{proposition_count_binary_rearrangements}\sidenote[][]{Den propositionen handlar om binära strängar, men vi har alfabetet $\{*,|\}$ -- det spelar så klart ingen roll för antalet vad vi kallar våra bokstäver.} hur man räknar antalet sådana ord -- våra ord har längd $n + k -1$ (de innehåller $n$ stjärnor och $k-1$ streck) och $k-1$ av dem är av ena typen, så det måste finnas totalt $\binom{n + k - 1}{k - 1}$ ord, och alltså lika många fördelningar.
   \end{proof}
 \end{proposition}
 
-\begin{example}
+\begin{example}% test
```

Comment:
Min redigering, för att det var lite tekniskt strul för mig och jag ville kolla om jag kunde redigera denna filen. Svaret var att jag kan det, men av någon anledning bara på hemsidan och inte lokalt?

Reply:
@vagdur  ska vi skicka in en ny pull request med redigeringarna?